aws/resource_aws_cloudwatch_metric_alarm_test.go

```diff
@@ -209,6 +229,26 @@ resource "aws_cloudwatch_metric_alarm" "foobar" {
 }`, rInt)
 }
 
+func testAccAWSCloudWatchMetricAlarmConfigDatapointsToAlarm(rName string) string {
+	return fmt.Sprintf(`
+resource "aws_cloudwatch_metric_alarm" "foobar" {
+  alarm_name                = "%s"
+  comparison_operator       = "GreaterThanOrEqualToThreshold"
+  datapoints_to_alarm       = "2"
+  evaluation_periods        = "2"
+  metric_name               = "CPUUtilization"
+  namespace                 = "AWS/EC2"
+	period                    = "120"
```

Comment:
Can you fix the spacing?

Reply:
Oi! I have since turned on whitespace character visibility to prevent this in the future. 😄